src/Arcus.Templates.Tests.Integration/WebApi/WebApiProject.cs

```diff
@@ -206,10 +206,19 @@ public static WebApiProject CreateNew(TestConfig configuration, WebApiProjectOpt
         /// </summary>
         public async Task StartAsync()
         {
-            Run(_configuration.BuildConfiguration,
-                TargetFramework.Net6_0,
-                CommandArgument.CreateOpen("ARCUS_HTTP_PORT", _baseUrl.Port));
+            var commandArguments = Array.Empty<CommandArgument>();
+            if (ContainsFile("appsettings.json"))
+            {
+                UpdateFileInProject(
+                    "appsettings.json", 
+                    contents => contents.Replace("\"ARCUS_HTTP_PORT\": 5000", $"\"ARCUS_HTTP_PORT\": {_baseUrl.Port}"));
```

Comment:
I don't think 5000 is a good idea to use as a custom port.
[Usage of port 5000](https://www.speedguide.net/port.php?port=5000)

It looks to be the default port that is used by Kestrel though.  Idk, maybe we should specify another port that exists in the range 1024 - 49151
https://www.sciencedirect.com/topics/computer-science/registered-port#

Just a suggestion.

Reply:
Yeah, that's probably why 5000 was used here, bc kestrel was using it.
But, actually, I don't have any preference here. Fine to use within that range.

Reply:
Changed to 4068.